### PR TITLE
Added my solution to Etch-a-Sketch project

### PR DESCRIPTION
**Pull request by morganbonhomme**
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

File: web_development_101/javascript_basics/project_etch_a_sketch.md [hunk 1]
```diff
@@ -43,8 +43,6 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
   <summary> Show Student Solutions </summary>
 
 - Add your solution below this line!
-* [Nick DiPreta's solution](https://github.com/NickDiPreta/etch-a-sketch) - [View in Browser](https://nickdipreta.github.io/etch-a-sketch/)
-* [justasdap Solution](https://github.com/JustasDap/etchasketch) - [View in Browser](https://htmlpreview.github.io/?https://github.com/JustasDap/etchasketch/blob/master/index.html)
```
Comment (ryanford-frontend) [inline web_development_101/javascript_basics/project_etch_a_sketch.md:47]: Why did you remove these?

Comment (morganbonhomme) [reply web_development_101/javascript_basics/project_etch_a_sketch.md:47]: Oops I didn't mean to do that... First time I'm contributing to an open source, it is a mistake... I just followed the video. 
Should I redo the process ? 
So sorry about that, that was not intended at all. I will pay more attention next time.

Comment (I3uckwheat) [reply web_development_101/javascript_basics/project_etch_a_sketch.md:47]: > Oops I didn't mean to do that... First time I'm contributing to an open source, it is a mistake... I just followed the video.
> Should I redo the process ?
> So sorry about that, that was not intended at all. I will pay more attention next time.

Yes, I will close this, you can just create a new one.
